server/admin/debug-clear.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
NOTE FOR REVIEWER: same functionality as before. just moved out of `server/index.js` and into its own file.